contrib/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+add_library(rapidjson INTERFACE)
```

Comment:
I believe you mean `fdbrpc` to `fdbclient`.
BTW, is `apiversioner.py` actually being used anywhere? There doesn't seem to be any code referencing it and it holds a lot of dated references to directories that no longer exist.

Reply:
No idea, I found it just by text searching rapidjson. @sfc-gh-ajbeamon do you know?

Reply:
It has been moved: if you look at the `Files changed`, there are a bunch of lines like `fdbclient/rapidjson/allocators.h → contrib/rapidjson/rapidjson/allocators.h`
FYI: the need for directory stutter, i.e. `contrib/rapidjson/rapidjson/document.h` is because rapidjson headers are included like this: `#include <rapidjson/document.h>`

Reply:
I see. Yes, it looks better so